2021-02-03 09:38:01 ERROR  : ОШИБКА:  значение не умещается в тип character(50)
2021-02-05 10:50:42 ERROR  : ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "pgs_antiair_pkey"
DETAIL:  Ключ "(n_sys)=(1)" уже существует.
2021-02-05 10:51:37 ERROR  : ОШИБКА:  текущая транзакция прервана, команды до конца блока транзакции игнорируются
2021-02-05 15:24:27 ERROR  : ОШИБКА:  колонка "res" не существует
LINE 1: ...T n_sys, lat, lon, ksa_level, ksa_type, ksa_call, res FROM p...
                                                             ^
2021-02-08 11:04:57 ERROR  : ОШИБКА:  колонка "res" не существует
LINE 1: ...T n_sys, lat, lon, ksa_level, ksa_type, ksa_call, res FROM p...
                                                             ^
2021-02-08 11:44:49 ERROR  : ОШИБКА:  колонка "res" в таблице "pgs_cmd_post" не существует
LINE 1: ...                                       $pgdescr$, res = 0 WH...
                                                             ^
2021-02-08 12:29:31 ERROR  : ОШИБКА:  текущая транзакция прервана, команды до конца блока транзакции игнорируются
2021-02-08 12:29:46 ERROR  : ОШИБКА:  колонка "Í" не существует
LINE 1: ...      $pgdescr$, res_0 = 0, res_1 = NULL, res_2 = Í, res_3 =...
                                                             ^
2021-02-08 12:30:59 ERROR  : ОШИБКА:  текущая транзакция прервана, команды до конца блока транзакции игнорируются
2021-02-08 12:31:08 ERROR  : ОШИБКА:  неверное значение для целого числа: "Í"
LINE 1: ...      $pgdescr$, res_0 = 0, res_1 = NULL, res_2 = 'Í', res_3...
                                                             ^
2021-02-08 14:41:33 ERROR  : ОШИБКА:  текущая транзакция прервана, команды до конца блока транзакции игнорируются
2021-02-09 10:10:48 ERROR  : ОШИБКА:  нулевое значение в колонке "n_sys" нарушает ограничение NOT NULL
DETAIL:  Ошибочная строка содержит (null, 0, 0, 1, 1, f, null, 1, null, null, null, null, null).
2021-02-15 10:05:13 ERROR  : ОШИБКА:  нулевое значение в колонке "n" нарушает ограничение NOT NULL
DETAIL:  Ошибочная строка содержит (2, null, 0, 0, 0, 0, 0, 0, 0).
2021-02-15 17:05:18 ERROR  : ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "pgs_aerodrome_runway_pkey"
DETAIL:  Ключ "(n_sys_aer, n)=(1, 1)" уже существует.
2021-02-15 17:05:22 ERROR  : ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "pgs_aerodrome_runway_pkey"
DETAIL:  Ключ "(n_sys_aer, n)=(1, 1)" уже существует.
2021-02-15 17:21:14 ERROR  : ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "pgs_aerodrome_runway_pkey"
DETAIL:  Ключ "(n_sys_aer, n)=(1, 1)" уже существует.
2021-02-15 17:21:47 ERROR  : ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "pgs_aerodrome_runway_pkey"
DETAIL:  Ключ "(n_sys_aer, n)=(1, 1)" уже существует.
2021-02-16 09:18:28 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "WHERE")
LINE 1: ... (q_runway, h_stop, d, cover_type, d_land, h_land WHERE n = ...
                                                             ^
2021-02-16 09:19:04 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "AND")
LINE 1: ... (q_runway, h_stop, d, cover_type, d_land, h_land AND n_sys_...
                                                             ^
2021-02-16 09:21:35 ERROR  : ОШИБКА:  нулевое значение в колонке "n" нарушает ограничение NOT NULL
DETAIL:  Ошибочная строка содержит (6, null, 0, 0, 0, 0, 0, 0, 4987).
2021-02-16 09:22:44 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: ")")
LINE 1: ...ds, h_land, d_land, aer_call, gp, h, num_runways, ) VALUES($...
                                                             ^
2021-02-16 09:26:48 ERROR  : ОШИБКА:  нулевое значение в колонке "n_sys_aer" нарушает ограничение NOT NULL
DETAIL:  Ошибочная строка содержит (null,      , 5.1, 5.2, 0, 0, 0, 0, 0, 0, 0, Re-re                                             , 0, 0, 0).
2021-02-16 09:27:46 ERROR  : ОШИБКА:  текущая транзакция прервана, команды до конца блока транзакции игнорируются
2021-02-16 09:28:55 ERROR  : ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "pgs_aerodromes_pkey"
DETAIL:  Ключ "(n_sys_aer)=(6)" уже существует.
2021-02-16 09:29:06 ERROR  : ОШИБКА:  текущая транзакция прервана, команды до конца блока транзакции игнорируются
2021-02-16 09:38:40 ERROR  : ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "pgs_aerodrome_runway_pkey"
DETAIL:  Ключ "(n_sys_aer, n)=(26, 0)" уже существует.
2021-02-16 09:41:10 ERROR  : ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "pgs_aerodromes_pkey"
DETAIL:  Ключ "(n_sys_aer)=(26)" уже существует.
2021-02-16 09:41:45 ERROR  : ОШИБКА:  отношение "pgs.pgs_aerodrome" не существует
LINE 4: DELETE FROM pgs.pgs_aerodrome
                    ^
2021-02-19 09:24:12 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "VALUE")
LINE 1: SELECT NEXT VALUE FOR pgs.pgs_aerodromes pgs_aerodromes_n_sy...
                    ^
2021-02-19 09:24:21 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "NEXT")
LINE 1: NEXT VALUE FOR pgs.pgs_aerodromes pgs_aerodromes_n_sys_aer_s...
        ^
2021-02-19 09:24:59 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "VALUE")
LINE 1: SELECT NEXT VALUE FOR pgs.pgs_aerodromes 'pgs_aerodromes_n_s...
                    ^
2021-02-19 09:27:29 ERROR  : ОШИБКА:  отношение "pgs_aerodromes_n_sys_aer_seq" не существует
LINE 1: SELECT nextval('pgs_aerodromes_n_sys_aer_seq')
                       ^
2021-02-19 09:27:51 ERROR  : ОШИБКА:  отношение "pgs_aerodromes_n_sys_aer_seq" не существует
LINE 1: SELECT nextval('pgs_aerodromes_n_sys_aer_seq')
                       ^
2021-02-19 09:30:34 ERROR  : ОШИБКА:  таблица "pgs" отсутствует в предложении FROM
LINE 1: SELECT nextval(pgs."pgs_aerodromes_n_sys_aer_seq")
                       ^
2021-02-19 10:11:16 ERROR  : ОШИБКА:  нулевое значение в колонке "n" нарушает ограничение NOT NULL
DETAIL:  Ошибочная строка содержит (32, null, 0, 0, 0, 0, 0, 0, 15078).
2021-03-03 10:23:16 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2021-03-03 10:23:55 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2021-03-03 10:25:31 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2021-03-03 17:07:26 ERROR  : ОШИБКА:  функция cardinality(integer[]) не существует
LINE 14: (select term, sum(weight)/cardinality(b) weight from ip_term...
                                   ^
HINT:  Функция с данными именем и типами аргументов не найдена. Возможно, вам следует добавить явные приведения типов.
2021-03-17 14:24:36 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "weight")
LINE 1: weight
        ^
2021-03-17 14:32:44 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "Function")
LINE 1: Function: dist2(integer, integer[])
        ^
2021-03-23 16:51:41 ERROR  : ОШИБКА:  колонка "res" не существует
LINE 1: ...T n_sys, lat, lon, ksa_level, ksa_type, ksa_call, res FROM p...
                                                             ^
2021-03-25 14:13:21 ERROR  : ОШИБКА:  колонка "res" не существует
LINE 1: ...a_call, res_0, res_1, res_2, res_3, res_4, res_5, res FROM p...
                                                             ^
2021-05-11 16:02:01 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2021-05-11 16:02:14 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:10:34 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:10:42 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:10:48 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:10:54 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:11:01 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:11:07 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:11:22 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:12:21 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:12:27 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:24:45 ERROR  : Error connecting to the server: authentication method 10 not supported

2022-04-06 12:24:59 ERROR  : Error connecting to the server: authentication method 10 not supported

2022-04-06 12:29:48 ERROR  : Error connecting to the server: authentication method 10 not supported

2022-04-06 12:30:39 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:36:52 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-04-06 12:44:33 ERROR  : Error connecting to the server: authentication method 10 not supported

2022-04-06 14:49:02 ERROR  : Error connecting to the server: authentication method 10 not supported

2022-04-06 14:54:46 ERROR  : ОШИБКА:  функция pg_last_xlog_receive_location() не существует
LINE 1: ...E NULL END as inrecovery, CASE WHEN usesuper THEN pg_last_xl...
                                                             ^
HINT:  Функция с данными именем и типами аргументов не найдена. Возможно, вам следует добавить явные приведения типов.
2022-04-06 14:54:48 ERROR  : Column not found in pgSet: usecreatedb
2022-04-06 14:54:49 ERROR  : Column not found in pgSet: usesuper
2022-04-06 14:54:49 ERROR  : Column not found in pgSet: upsince
2022-04-06 14:54:50 ERROR  : Column not found in pgSet: confloadedsince
2022-04-06 14:54:50 ERROR  : Column not found in pgSet: inrecovery
2022-04-06 14:54:50 ERROR  : Column not found in pgSet: replayloc
2022-04-06 14:54:50 ERROR  : Column not found in pgSet: receiveloc
2022-04-06 14:54:51 ERROR  : Column not found in pgSet: replay_timestamp
2022-04-06 14:54:51 ERROR  : Column not found in pgSet: isreplaypaused
2022-04-06 14:55:00 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-04-06 14:55:02 ERROR  : ОШИБКА:  столбец rel.relhasoids не существует
LINE 1: ...t_userbyid(rel.relowner) AS relowner, rel.relacl, rel.relhas...
                                                             ^
2022-04-06 14:55:03 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-04-06 14:55:18 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-04-06 14:55:20 ERROR  : ОШИБКА:  столбец rel.relhasoids не существует
LINE 1: ...t_userbyid(rel.relowner) AS relowner, rel.relacl, rel.relhas...
                                                             ^
2022-04-06 14:55:20 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-04-06 14:55:23 ERROR  : ОШИБКА:  столбец rel.relhasoids не существует
LINE 1: ...t_userbyid(rel.relowner) AS relowner, rel.relacl, rel.relhas...
                                                             ^
2022-04-27 15:09:07 ERROR  : ОШИБКА:  функция pg_last_xlog_receive_location() не существует
LINE 1: ...E NULL END as inrecovery, CASE WHEN usesuper THEN pg_last_xl...
                                                             ^
HINT:  Функция с данными именем и типами аргументов не найдена. Возможно, вам следует добавить явные приведения типов.
2022-04-27 15:09:08 ERROR  : Column not found in pgSet: usecreatedb
2022-04-27 15:09:09 ERROR  : Column not found in pgSet: usesuper
2022-04-27 15:09:09 ERROR  : Column not found in pgSet: upsince
2022-04-27 15:09:10 ERROR  : Column not found in pgSet: confloadedsince
2022-04-27 15:09:10 ERROR  : Column not found in pgSet: inrecovery
2022-04-27 15:09:10 ERROR  : Column not found in pgSet: replayloc
2022-04-27 15:09:10 ERROR  : Column not found in pgSet: receiveloc
2022-04-27 15:09:10 ERROR  : Column not found in pgSet: replay_timestamp
2022-04-27 15:09:10 ERROR  : Column not found in pgSet: isreplaypaused
2022-04-27 15:09:22 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-04-27 15:09:24 ERROR  : ОШИБКА:  столбец rel.relhasoids не существует
LINE 1: ...t_userbyid(rel.relowner) AS relowner, rel.relacl, rel.relhas...
                                                             ^
2022-04-27 15:09:25 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-04-27 15:09:29 ERROR  : ОШИБКА:  столбец rel.relhasoids не существует
LINE 1: ...t_userbyid(rel.relowner) AS relowner, rel.relacl, rel.relhas...
                                                             ^
2022-04-27 15:09:40 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-04-27 15:09:42 ERROR  : ОШИБКА:  столбец rel.relhasoids не существует
LINE 1: ...t_userbyid(rel.relowner) AS relowner, rel.relacl, rel.relhas...
                                                             ^
2022-04-27 15:09:42 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-05-04 12:49:16 ERROR  : ОШИБКА:  функция pg_last_xlog_receive_location() не существует
LINE 1: ...E NULL END as inrecovery, CASE WHEN usesuper THEN pg_last_xl...
                                                             ^
HINT:  Функция с данными именем и типами аргументов не найдена. Возможно, вам следует добавить явные приведения типов.
2022-05-04 12:49:17 ERROR  : Column not found in pgSet: usecreatedb
2022-05-04 12:49:19 ERROR  : Column not found in pgSet: usesuper
2022-05-04 12:49:19 ERROR  : Column not found in pgSet: upsince
2022-05-04 12:49:20 ERROR  : Column not found in pgSet: confloadedsince
2022-05-04 12:49:20 ERROR  : Column not found in pgSet: inrecovery
2022-05-04 12:49:20 ERROR  : Column not found in pgSet: replayloc
2022-05-04 12:49:20 ERROR  : Column not found in pgSet: receiveloc
2022-05-04 12:49:21 ERROR  : Column not found in pgSet: replay_timestamp
2022-05-04 12:49:21 ERROR  : Column not found in pgSet: isreplaypaused
2022-05-18 12:26:29 ERROR  : ОШИБКА:  функция pg_last_xlog_receive_location() не существует
LINE 1: ...E NULL END as inrecovery, CASE WHEN usesuper THEN pg_last_xl...
                                                             ^
HINT:  Функция с данными именем и типами аргументов не найдена. Возможно, вам следует добавить явные приведения типов.
2022-05-18 12:26:31 ERROR  : Column not found in pgSet: usecreatedb
2022-05-18 12:26:31 ERROR  : Column not found in pgSet: usesuper
2022-05-18 12:26:32 ERROR  : Column not found in pgSet: upsince
2022-05-18 12:26:32 ERROR  : Column not found in pgSet: confloadedsince
2022-05-18 12:26:32 ERROR  : Column not found in pgSet: inrecovery
2022-05-18 12:26:33 ERROR  : Column not found in pgSet: replayloc
2022-05-18 12:26:33 ERROR  : Column not found in pgSet: receiveloc
2022-05-18 12:26:33 ERROR  : Column not found in pgSet: replay_timestamp
2022-05-18 12:26:33 ERROR  : Column not found in pgSet: isreplaypaused
2022-05-18 12:26:58 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-05-18 12:27:01 ERROR  : ОШИБКА:  столбец rel.relhasoids не существует
LINE 1: ...t_userbyid(rel.relowner) AS relowner, rel.relacl, rel.relhas...
                                                             ^
2022-05-18 12:27:02 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-05-18 12:31:59 ERROR  : ОШИБКА:  функция pg_last_xlog_receive_location() не существует
LINE 1: ...E NULL END as inrecovery, CASE WHEN usesuper THEN pg_last_xl...
                                                             ^
HINT:  Функция с данными именем и типами аргументов не найдена. Возможно, вам следует добавить явные приведения типов.
2022-05-18 12:32:01 ERROR  : Column not found in pgSet: usecreatedb
2022-05-18 12:32:03 ERROR  : Column not found in pgSet: usesuper
2022-05-18 12:32:03 ERROR  : Column not found in pgSet: upsince
2022-05-18 12:32:03 ERROR  : Column not found in pgSet: confloadedsince
2022-05-18 12:32:04 ERROR  : Column not found in pgSet: inrecovery
2022-05-18 12:32:04 ERROR  : Column not found in pgSet: replayloc
2022-05-18 12:32:04 ERROR  : Column not found in pgSet: receiveloc
2022-05-18 12:32:04 ERROR  : Column not found in pgSet: replay_timestamp
2022-05-18 12:32:04 ERROR  : Column not found in pgSet: isreplaypaused
2022-05-20 11:43:51 ERROR  : ОШИБКА:  функция pg_last_xlog_receive_location() не существует
LINE 1: ...E NULL END as inrecovery, CASE WHEN usesuper THEN pg_last_xl...
                                                             ^
HINT:  Функция с данными именем и типами аргументов не найдена. Возможно, вам следует добавить явные приведения типов.
2022-05-20 11:43:53 ERROR  : Column not found in pgSet: usecreatedb
2022-05-20 11:43:54 ERROR  : Column not found in pgSet: usesuper
2022-05-20 11:43:55 ERROR  : Column not found in pgSet: upsince
2022-05-20 11:43:55 ERROR  : Column not found in pgSet: confloadedsince
2022-05-20 11:43:55 ERROR  : Column not found in pgSet: inrecovery
2022-05-20 11:43:55 ERROR  : Column not found in pgSet: replayloc
2022-05-20 11:43:55 ERROR  : Column not found in pgSet: receiveloc
2022-05-20 11:43:55 ERROR  : Column not found in pgSet: replay_timestamp
2022-05-20 11:43:55 ERROR  : Column not found in pgSet: isreplaypaused
2022-05-20 11:44:05 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-05-20 11:44:10 ERROR  : ОШИБКА:  столбец rel.relhasoids не существует
LINE 1: ...t_userbyid(rel.relowner) AS relowner, rel.relacl, rel.relhas...
                                                             ^
2022-05-20 11:44:11 ERROR  : ОШИБКА:  столбец "proisagg" не существует
LINE 9:  WHERE proisagg = FALSE AND pronamespace = 2200::oid
               ^
HINT:  Возможно, предполагалась ссылка на столбец "pr.prolang".
2022-05-20 11:46:16 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-05-20 11:46:26 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-05-20 12:55:00 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "ax_assignment")
LINE 6:             WHERE ax_page.id = -14 ax_assignment.status_code...
                                           ^
2022-05-20 15:38:11 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "$")
LINE 4:             WHERE ax_page.id = $page_id AND ax_assignment.st...
                                       ^
2022-05-27 11:21:57 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-05-27 11:22:08 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-05-27 11:22:22 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-05-27 11:22:29 ERROR  : Error connecting to the server: ВАЖНО:  пользователь "postgres" не прошёл проверку подлинности (по паролю)

2022-06-01 13:46:39 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "\")
LINE 1: SELECT first_name || \' \' || middle_name fio from students ...
                             ^
2022-06-01 14:03:40 ERROR  : ОШИБКА:  отношение "students_to_group" не существует
LINE 3: INNER JOIN students_to_group ON students_to_group.student_id...
                   ^
2022-06-01 14:04:31 ERROR  : ОШИБКА:  неоднозначная ссылка на столбец "id"
LINE 6: where id = -7;
              ^
2022-06-01 14:04:50 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "students")
LINE 1: students.
        ^
2022-06-01 14:16:45 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "CASE")
LINE 4:         CASE WHEN groups.name is not null groups.name 
                ^
2022-06-01 14:17:23 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "CASE")
LINE 4:         CASE WHEN groups.name is not null groups.name 
                ^
2022-06-01 14:17:32 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "CASE")
LINE 4:         CASE WHEN groups.name is not null groups.name 
                ^
2022-06-01 14:18:26 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "groups")
LINE 4:         CASE WHEN groups.name is not null groups.name 
                                                  ^
2022-06-01 14:19:48 ERROR  : ОШИБКА:  в конструкции CASE типы text и integer не имеют общего
LINE 5: ...EN ax_settings.notification_type is not null then ax_setting...
                                                             ^
2022-06-01 14:33:21 ERROR  : ОШИБКА:  в конструкции CASE типы text и boolean не имеют общего
LINE 6: ...    WHEN ax_settings.monaco_dark is not null then ax_setting...
                                                             ^
2022-06-01 14:35:43 ERROR  : ОШИБКА:  функция to_char(boolean, unknown) не существует
LINE 6: ...    WHEN ax_settings.monaco_dark is not null then to_char(ax...
                                                             ^
HINT:  Функция с данными именем и типами аргументов не найдена. Возможно, вам следует добавить явные приведения типов.
2022-06-01 14:38:22 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "CASE")
LINE 3:         CASE WHEN groups.name is not null then groups.name 
                ^
2022-06-01 14:41:28 ERROR  : ОШИБКА:  таблица "ax_settings" отсутствует в предложении FROM
LINE 2:             groups.name, ax_settings.email, ax_settings.noti...
                                 ^
2022-06-01 14:43:35 ERROR  : ОШИБКА:  в конструкции CASE типы text и integer не имеют общего
LINE 5: ...EN ax_settings.notification_type is not null then ax_setting...
                                                             ^
2022-08-01 14:05:02 ERROR  : ОШИБКА:  функция pg_last_xlog_receive_location() не существует
LINE 1: ...E NULL END as inrecovery, CASE WHEN usesuper THEN pg_last_xl...
                                                             ^
HINT:  Функция с данными именем и типами аргументов не найдена. Возможно, вам следует добавить явные приведения типов.
2022-08-01 14:05:03 ERROR  : Column not found in pgSet: usecreatedb
2022-08-01 14:05:07 ERROR  : Column not found in pgSet: usesuper
2022-08-01 14:05:07 ERROR  : Column not found in pgSet: upsince
2022-08-01 14:05:07 ERROR  : Column not found in pgSet: confloadedsince
2022-08-01 14:05:08 ERROR  : Column not found in pgSet: inrecovery
2022-08-01 14:05:08 ERROR  : Column not found in pgSet: replayloc
2022-08-01 14:05:09 ERROR  : Column not found in pgSet: receiveloc
2022-08-01 14:05:09 ERROR  : Column not found in pgSet: replay_timestamp
2022-08-01 14:05:09 ERROR  : Column not found in pgSet: isreplaypaused
2022-08-02 16:44:39 ERROR  : ОШИБКА:  в элементе предложения FROM неверная ссылка на таблицу "ax_message"
LINE 26:             WHERE ax_task.page_id = -13 AND (ax_message.type...
                                                      ^
HINT:  Возможно, предполагалась ссылка на псевдоним таблицы "m1".
2022-08-02 16:45:09 ERROR  : ОШИБКА:  в элементе предложения FROM неверная ссылка на таблицу "ax_message"
LINE 26: ...  WHERE ax_task.page_id = -13 AND (m1.type = 2 OR ax_message...
                                                              ^
HINT:  Возможно, предполагалась ссылка на псевдоним таблицы "m1".
2022-08-03 16:07:02 ERROR  : ОШИБКА:  выражение ORDER BY "id" неоднозначно
LINE 3:         WHERE page_id = -13 AND status = 1 ORDER BY id;
                                                            ^
2022-08-03 16:58:54 ERROR  : ОШИБКА:  неоднозначная ссылка на столбец "id"
LINE 8:             WHERE id = -5 AND ax_assignment.status_code = 5 ...
                          ^
2022-08-03 16:58:57 ERROR  : ОШИБКА:  неоднозначная ссылка на столбец "id"
LINE 8:             WHERE id = -5 AND ax_assignment.status_code = 5 ...
                          ^
2022-08-04 12:12:24 ERROR  : ОШИБКА:  неверное значение для целого числа: "$student_id"
LINE 5: ...s = 0 AND ax_assignment_student.student_user_id = '$student_...
                                                             ^
2022-08-04 12:21:27 ERROR  : ОШИБКА:  неверное значение для целого числа: "$page_id"
LINE 26:             WHERE ax_task.page_id = '$page_id' ORDER BY mid ...
                                             ^
2022-08-04 14:11:56 ERROR  : ОШИБКА:  неверное значение для целого числа: "$page_id"
LINE 26:             WHERE ax_task.page_id = '$page_id' ORDER BY mid ...
                                             ^
2022-08-04 14:22:18 ERROR  : ОШИБКА:  столбец "atype" не существует
LINE 26:             WHERE ax_task.page_id = -13 AND atype != null OR...
                                                     ^
2022-08-04 14:22:41 ERROR  : ОШИБКА:  неоднозначная ссылка на столбец "type"
LINE 26:             WHERE ax_task.page_id = -13 AND type != null ORD...
                                                     ^
2022-08-04 14:26:42 ERROR  : ОШИБКА:  неверное значение для целого числа: "$page_id"
LINE 5:         WHERE page_id = '$page_id' AND status = '$status' AN...
                                ^
2022-08-04 14:33:31 ERROR  : ОШИБКА:  незавершённый идентификатор в кавычках (примерное положение: "";")
LINE 8:                 ORDER BY grp, fio";
                                         ^
2022-08-04 15:20:28 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "ORDER")
LINE 26:             WHERE ax_task.page_id = -13 AND ORDER BY mid DES...
                                                     ^
2022-08-15 11:29:22 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "$")
LINE 9:             WHERE ax_page_prep.prep_user_id = $teacher_id AN...
                                                      ^
2022-08-15 11:42:00 ERROR  : ОШИБКА:  таблица "ax_mesage" отсутствует в предложении FROM
LINE 9:             INNER JOIN ax_message ON ax_mesage.assignment_id...
                                             ^
2022-08-15 11:42:41 ERROR  : ОШИБКА:  столбец ax_message.title не существует
LINE 2: ...atus_code, ax_assignment_student.student_user_id, ax_message...
                                                             ^
2022-08-15 11:51:38 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "JOIN")
LINE 10:             INNER JOIN students ON students.id = ax_message....
                           ^
2022-08-15 11:59:51 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: ".")
LINE 3:             students.middle_name, students.first_name FROM a...
                            ^
2022-08-15 12:01:09 ERROR  : ОШИБКА:  таблица "ax_message" отсутствует в предложении FROM
LINE 8:             INNER JOIN students ON students.id = ax_message....
                                                         ^
2022-08-15 12:01:18 ERROR  : ОШИБКА:  таблица "students" отсутствует в предложении FROM
LINE 3:             students.middle_name, students.first_name FROM a...
                    ^
2022-08-15 12:02:16 ERROR  : ОШИБКА:  таблица "ax_message" отсутствует в предложении FROM
LINE 9: ...         WHERE ax_page_prep.prep_user_id = -5 AND ax_message...
                                                             ^
2022-08-15 12:07:26 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: ".")
LINE 3: students.middle_name, students.first_name FROM ax_task
                ^
2022-08-15 12:08:56 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: ")")
LINE 11: ...ND ax_message.status = 0) OR ax_assignment.status_code = 5);
                                                                      ^
2022-08-15 12:11:25 ERROR  : ОШИБКА:  таблица "m1" отсутствует в предложении FROM
LINE 11:             LEFT JOIN students s2 ON s2.id = m1.sender_user_...
                                                      ^
2022-08-15 12:11:36 ERROR  : ОШИБКА:  в элементе предложения FROM неверная ссылка на таблицу "students"
LINE 3: students.middle_name, students.first_name FROM ax_task
        ^
HINT:  Возможно, предполагалась ссылка на псевдоним таблицы "s1".
2022-08-15 12:15:49 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "$")
LINE 5: ...    WHERE ax_assignment_student.student_user_id = $student_i...
                                                             ^
2022-08-15 12:20:04 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: ".")
LINE 1: ...ment.status_code, teachers.first_name ' ' teachers.last_name...
                                                             ^
2022-08-15 12:23:41 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "ax_message")
LINE 8: ...udent.student_user_id = -7 AND ax_page.status = 1 ax_message...
                                                             ^
2022-08-15 14:42:43 ERROR  : ОШИБКА:  аргумент конструкции AND должен иметь логический тип, а не integer
LINE 1: UPDATE ax_assignment SET status_code = 2 AND status_text = "...
                                               ^
2022-08-15 14:42:59 ERROR  : ОШИБКА:  столбец "активно" не существует
LINE 1: ... ax_assignment SET status_code = 2, status_text = "активно" ...
                                                             ^
2022-08-15 14:43:19 ERROR  : ОШИБКА:  столбец "ожидает проверки" не существует
LINE 2: ... ax_assignment SET status_code = 5, status_text = "ожидает п...
                                                             ^
2022-08-17 14:28:38 ERROR  : ОШИБКА:  в элементе предложения FROM неверная ссылка на таблицу "groups"
LINE 2:               INNER JOIN ax_page_group ON groups.id = ax_pag...
                                                  ^
HINT:  Возможно, предполагалась ссылка на псевдоним таблицы "g".
2022-08-17 14:28:52 ERROR  : ОШИБКА:  в элементе предложения FROM неверная ссылка на таблицу "groups"
LINE 2:               INNER JOIN ax_page_group ON groups.id = ax_pag...
                                                  ^
HINT:  Возможно, предполагалась ссылка на псевдоним таблицы "g".
2022-08-17 14:29:45 ERROR  : ОШИБКА:  таблица "group" отсутствует в предложении FROM
LINE 3: ...          INNER JOIN students_to_groups ON g.id = students_t...
                                                             ^
2022-09-09 16:27:20 ERROR  : ОШИБКА:  столбец ax_assignment_student.student_id не существует
LINE 8:               INNER JOIN ax_assignment_student ON ax_assignm...
                                                          ^
2022-09-13 11:05:20 ERROR  : ОШИБКА:  неверное значение для целого числа: "$page_id"
LINE 6:               WHERE page_id ='$page_id' ORDER BY g.id, fi;
                                     ^
2022-09-13 11:40:09 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "(")
LINE 2: ...ax_task.id = -16 THEN ax_task.id ELSE null END max(task_id) 
                                                             ^
2022-09-13 11:42:05 ERROR  : ОШИБКА:  столбец "s.id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT DISTINCT s.id, s.middle_name || ' ' || s.first_name a...
                        ^
2022-09-13 11:46:18 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: ",")
LINE 2: ...k.id = -16 THEN ax_task.id ELSE null END task_id, , max(task...
                                                             ^
2022-09-13 11:46:26 ERROR  : ОШИБКА:  столбец "s.id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT DISTINCT s.id, s.middle_name || ' ' || s.first_name a...
                        ^
2022-09-13 11:47:34 ERROR  : ОШИБКА:  выражения SELECT DISTINCT ON должны соответствовать начальным выражениям ORDER BY
LINE 1: SELECT DISTINCT ON (s.id) s.id, s.middle_name || ' ' || s.fi...
                            ^
2022-09-13 11:47:48 ERROR  : ОШИБКА:  выражения SELECT DISTINCT ON должны соответствовать начальным выражениям ORDER BY
LINE 1: SELECT DISTINCT ON (s.id) s.id, s.middle_name || ' ' || s.fi...
                            ^
2022-09-13 11:48:08 ERROR  : ОШИБКА:  столбец "s.id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT DISTINCT s.id, s.middle_name || ' ' || s.first_name a...
                        ^
2022-09-13 11:48:44 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "(")
LINE 2: ... ax_task.id = -16 THEN ax_task.id ELSE null END max(task_id)
                                                              ^
2022-09-13 11:51:26 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "DISTINCT")
LINE 2: ...EN ax_task.id = -16 THEN ax_task.id ELSE null END DISTINCT t...
                                                             ^
2022-09-13 11:51:34 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "DISTINCT")
LINE 1: ...i, s.login, g.id as group_id, g.name as group_name, DISTINCT
                                                               ^
2022-09-13 11:52:29 ERROR  : ОШИБКА:  столбец "s.id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT DISTINCT s.id, s.middle_name || ' ' || s.first_name a...
                        ^
2022-09-13 11:52:59 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "SELECT")
LINE 12:               WHERE ax_page_group.page_id = -13 SELECT MAX(t...
                                                         ^
2022-09-13 11:53:21 ERROR  : ОШИБКА:  агрегатные функции нельзя применять в конструкции WHERE
LINE 12: ...ax_page_group.page_id = -13 AND task_id = (SELECT MAX(task_i...
                                                              ^
2022-09-13 11:56:01 ERROR  : ОШИБКА:  столбец "s.id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT DISTINCT s.id, s.middle_name || ' ' || s.first_name a...
                        ^
2022-09-13 11:56:33 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "GROUP")
LINE 13:               GROUP BY s.id;
                       ^
2022-09-13 11:56:49 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "GROUP")
LINE 12: ...ERE ax_page_group.page_id = -13 ORDER BY g.id, fi GROUP BY s...
                                                              ^
2022-09-13 11:57:16 ERROR  : ОШИБКА:  столбец "s.id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT DISTINCT s.id, s.middle_name || ' ' || s.first_name a...
                        ^
2022-09-13 11:57:20 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "INNER")
LINE 5:               INNER JOIN ax_page_group ON g.id = ax_page_gro...
                      ^
2022-09-13 12:02:24 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "(")
LINE 2: ... ax_task.id = -16 THEN ax_task.id ELSE null END MAX(task_id)
                                                              ^
2022-09-13 12:02:33 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "(")
LINE 2: ...ax_task.id = -16 THEN ax_task.id ELSE null END MAX (task_id)
                                                              ^
2022-09-13 12:06:42 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "FROM")
LINE 3:               FROM groups g
                      ^
2022-09-13 12:15:38 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "task_id")
LINE 2: ...WHEN ax_task.id = -16 THEN ax_task.id ELSE null END task_id)
                                                               ^
2022-09-13 12:15:50 ERROR  : ОШИБКА:  в конструкции SELECT DISTINCT выражения ORDER BY должны быть в списке выборки
LINE 12: ...HERE ax_page_group.page_id = -13 ORDER BY g.id, fi, task_id;
                                                                ^
2022-09-13 12:15:58 ERROR  : ОШИБКА:  столбец "s.id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT DISTINCT s.id, s.middle_name || ' ' || s.first_name a...
                        ^
2022-09-13 12:17:39 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "WHERE")
LINE 14:               WHERE ax_page_group.page_id = -13 ORDER BY g.i...
                       ^
2022-09-13 12:23:42 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "WHERE")
LINE 14:               WHERE ax_page_group.page_id = -13 AND MAX(task...
                       ^
2022-09-13 12:25:41 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "WHERE")
LINE 14:               WHERE ax_page_group.page_id = -13 ORDER BY g.i...
                       ^
2022-09-13 12:33:07 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: ",")
LINE 1: SELECT DISTINCT ON (s.id), s.middle_name || ' ' || s.first_n...
                                 ^
2022-09-13 12:33:20 ERROR  : ОШИБКА:  неоднозначная ссылка на столбец "id"
LINE 1: SELECT DISTINCT ON (s.id) id, s.middle_name || ' ' || s.firs...
                                  ^
2022-09-13 12:33:23 ERROR  : ОШИБКА:  выражения SELECT DISTINCT ON должны соответствовать начальным выражениям ORDER BY
LINE 1: SELECT DISTINCT ON (s.id) s.id, s.middle_name || ' ' || s.fi...
                            ^
2022-09-13 12:34:20 ERROR  : ОШИБКА:  выражения SELECT DISTINCT ON должны соответствовать начальным выражениям ORDER BY
LINE 1: SELECT DISTINCT ON (s.id) s.id, s.middle_name || ' ' || s.fi...
                            ^
2022-09-13 12:37:29 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "DISTINCT")
LINE 1: ...k.id = -16 THEN ax_task.id ELSE null END task_id, DISTINCT s...
                                                             ^
2022-09-13 12:43:14 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: ".")
LINE 1: SELECT DISTINCT (s.id) s.id, s.middle_name || ' ' || s.first...
                                ^
2022-09-13 17:10:22 ERROR  : ОШИБКА:  выражения SELECT DISTINCT ON должны соответствовать начальным выражениям ORDER BY
LINE 1: SELECT DISTINCT ON (s.id) s.id, s.middle_name || ' ' || s.fi...
                            ^
2022-09-13 17:14:44 ERROR  : ОШИБКА:  подзапрос во FROM должен иметь псевдоним
LINE 3: FROM( 
            ^
HINT:  Например, FROM (SELECT ...) [AS] foo.
2022-09-20 10:13:50 ERROR  : ОШИБКА:  отношение "ax_task_files" не существует
LINE 1: DELETE FROM ax_task_files 
                    ^
2022-09-20 10:14:15 ERROR  : ОШИБКА:  столбец ax_task.task_id не существует
LINE 4:                 WHERE ax_task.task_id = 6;
                              ^
HINT:  Возможно, предполагалась ссылка на столбец "ax_task.page_id".
2022-09-20 10:14:48 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "id")
LINE 1: id
        ^
2022-09-27 15:26:50 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "CASE")
LINE 5:     CASE WHEN students.id = -7 THEN 1 ELSE 0 END status
            ^
2022-09-27 15:27:04 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "CASE")
LINE 5:     CASE WHEN students.id = -7 THEN 1 ELSE 0 END status
            ^
2022-09-27 15:27:13 ERROR  : ОШИБКА:  столбец a.fi не существует
LINE 1: SELECT a.id, a.fi, a.login, a.group_id, a.group_name, max(a....
                     ^
2022-09-27 15:27:42 ERROR  : ОШИБКА:  неоднозначная ссылка на столбец "status"
LINE 1: SELECT a.id, a.page_id, a.title, a.status, a.assignment_id, ...
                                         ^
2022-09-27 15:29:23 ERROR  : ОШИБКА:  столбец "a.assignment_student_status" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: ...nish_limit, a.status_code, a.mark, a.status_text, a.assignme...
                                                             ^
2022-09-27 15:44:50 ERROR  : ОШИБКА:  неверное значение для целого числа: "$page_id"
LINE 26:         WHERE ax_task.page_id = '$page_id' AND (m1.type = 0 ...
                                         ^
2022-09-27 15:51:32 ERROR  : ОШИБКА:  столбец "amark" не существует
LINE 26: ...  WHERE ax_task.page_id = -13 AND m1.type = 1 AND amark = ''...
                                                              ^
HINT:  Возможно, предполагалась ссылка на столбец "ax_assignment.mark".
2022-09-27 15:52:26 ERROR  : ОШИБКА:  оператор не существует: text < integer
LINE 26: ...e_id = -13 AND m1.type = 1 AND ax_assignment.mark <1 ORDER B...
                                                              ^
HINT:  Оператор с данными именем и типами аргументов не найден. Возможно, вам следует добавить явные приведения типов.
2022-09-27 15:53:58 ERROR  : ОШИБКА:  пустой идентификатор в кавычках (примерное положение: """")
LINE 26: ...id = -13 AND m1.type = 1 AND ax_assignment.mark = "" ORDER B...
                                                              ^
2022-09-27 15:54:15 ERROR  : ОШИБКА:  оператор не существует: ! text
LINE 26: ...  WHERE ax_task.page_id = -13 AND m1.type = 1 AND !ax_assign...
                                                              ^
HINT:  Оператор с данными именем и типами аргументов не найден. Возможно, вам следует добавить явные приведения типов.
2022-09-27 15:54:32 ERROR  : ОШИБКА:  оператор не существует: text <> integer
LINE 26: ..._id = -13 AND m1.type = 1 AND (ax_assignment.mark != 5) ORDE...
                                                              ^
HINT:  Оператор с данными именем и типами аргументов не найден. Возможно, вам следует добавить явные приведения типов.
2022-09-27 15:56:27 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "ax_task")
LINE 2:         ax_task.id tid, ax_assignment.id aid, m1.id mid, ax_...
                ^
2022-09-27 15:59:19 ERROR  : ОШИБКА:  столбец "s1.middle_name" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT DISTINCT s1.middle_name || ' ' || s1.first_name fio, ...
                        ^
2022-09-27 15:59:29 ERROR  : ОШИБКА:  столбец "s1.middle_name" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT s1.middle_name || ' ' || s1.first_name fio, groups.na...
               ^
2022-09-27 16:00:21 ERROR  : ОШИБКА:  столбец "groups.name" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: ...TINCT s1.middle_name || ' ' || s1.first_name fio, groups.nam...
                                                             ^
2022-09-27 16:01:59 ERROR  : ОШИБКА:  агрегатные функции нельзя применять в конструкции GROUP BY
LINE 2:         ax_task.id tid, ax_assignment.id aid, max(m1.id) mid...
                                                      ^
2022-09-27 16:02:07 ERROR  : ОШИБКА:  агрегатные функции нельзя применять в конструкции GROUP BY
LINE 2:         ax_task.id tid, ax_assignment.id aid, max(m1.id) mid...
                                                      ^
2022-09-27 16:02:17 ERROR  : ОШИБКА:  неоднозначная ссылка на столбец "type"
LINE 27:  GROUP BY fio, grp, gid, tid, aid, sid, fid, type
                                                      ^
2022-09-27 16:02:28 ERROR  : ОШИБКА:  столбец "m1.type" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 3:         ax_message_attachment.id fid, m1.type,
                                              ^
2022-09-27 16:02:41 ERROR  : ОШИБКА:  столбец "m1.sender_user_type" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 7:         WHEN m1.sender_user_type = 0 then '?' 
                     ^
2022-09-27 16:03:03 ERROR  : ОШИБКА:  столбец "m1.date_time" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 11:         ax_assignment.status_text astext, to_char(m1.date_ti...
                                                           ^
2022-09-27 16:05:04 ERROR  : ОШИБКА:  столбец "m1.date_time" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 11:         ax_assignment.status_text astext, to_char(m1.date_ti...
                                                           ^
2022-09-27 16:05:16 ERROR  : ОШИБКА:  столбец "m1.full_text" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 12:         m1.full_text mtext, m1.sender_user_type mtype, m1.st...
                 ^
2022-09-27 16:05:34 ERROR  : ОШИБКА:  столбец "m1.sender_user_type" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 12:         m1.full_text mtext, m1.sender_user_type mtype, m1.st...
                                     ^
2022-09-27 16:05:48 ERROR  : ОШИБКА:  столбец "m1.status" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 12: ...   m1.full_text mtext, m1.sender_user_type mtype, m1.status ...
                                                              ^
2022-09-27 16:07:19 ERROR  : ОШИБКА:  столбец "m2.full_text" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 12: ...xt, m1.sender_user_type mtype, m1.status mstatus, m2.full_te...
                                                              ^
2022-09-27 16:10:31 ERROR  : ОШИБКА:  столбец "ax_assignment_student.student_user_id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 2: ...ask.id tid, ax_assignment.id aid, max(m1.id) mid, ax_assignm...
                                                             ^
2022-09-27 16:38:40 ERROR  : ОШИБКА:  ошибка синтаксиса (примерное положение: "FROM")
LINE 4:         FROM ax_task 
                ^
2022-09-27 16:38:56 ERROR  : ОШИБКА:  столбец "val" не существует
LINE 16:  GROUP BY fio, grp, gid, tid, aid, sid, m1.type, val, title,...
                                                          ^
2022-09-27 16:42:04 ERROR  : ОШИБКА:  столбец "mtime" не существует
LINE 26: ...io, grp, gid, tid, aid, sid, m1.type, val, title, mtime, mte...
                                                              ^
2022-09-27 16:53:09 ERROR  : ОШИБКА:  таблица "s1" отсутствует в предложении FROM
LINE 1: SELECT s1.middle_name || ' ' || s1.first_name fio, groups.na...
               ^
2022-09-27 16:53:37 ERROR  : ОШИБКА:  столбец "s1.middle_name" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT s1.middle_name || ' ' || s1.first_name fio, groups.na...
               ^
2022-09-27 16:53:58 ERROR  : ОШИБКА:  столбец "s1.middle_name" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 1: SELECT DISTINCT s1.middle_name || ' ' || s1.first_name fio, ...
                        ^
2022-09-27 17:03:41 ERROR  : ОШИБКА:  оператор не существует: integer = record
LINE 20: ...dent_user_id or m1.sender_user_type=1) AND m1.status = (0,1)
                                                                 ^
HINT:  Оператор с данными именем и типами аргументов не найден. Возможно, вам следует добавить явные приведения типов.
2022-09-30 15:59:26 ERROR  : ОШИБКА:  столбец "ax_message.id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 3:           ORDER BY id DESC
                           ^
2022-09-30 15:59:32 ERROR  : ОШИБКА:  столбец "ax_message.id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 3:           ORDER BY id 
                           ^
2022-09-30 16:00:34 ERROR  : ОШИБКА:  столбец "ax_message.id" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
LINE 4:           ORDER BY id 
                           ^
